framer/Animation.coffee

```diff
@@ -190,7 +190,11 @@ class exports.Animation extends EventEmitter
 	_updateValue: (value) =>
 
 		for k, v of @_stateB
-			@_target[k] = Utils.mapRange(value, 0, 1, @_stateA[k], @_stateB[k])
+
+			if v instanceof Color or @_stateA[k] instanceof Color
+				@_target[k] = Color.blend(@_stateA[k], @_stateB[k], value)
+			else
+				@_target[k] = Utils.mapRange(value, 0, 1, @_stateA[k], @_stateB[k])
```

Comment:
mapRange is more performant (important for animations) but a little bit harder to use. I think we should stick to mapRange (private api)